packages-exp/auth-compat-exp/package.json

```diff
@@ -41,8 +42,8 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@firebase/app-compat": "0.x",
-    "rollup": "2.35.1",
     "@rollup/plugin-json": "4.1.0",
+    "rollup": "^2.35.1",
```

Comment:
revert this

Reply:
I believe we need this now that there's a separate rollup config for the webdriver tests. The main package has it as well